dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/DatasetReportService.java

```diff
@@ -173,6 +174,15 @@ private String getTags(FileMetadata fileMetadata, DataFile dataFile) {
         return tags;
     }
 
+    private String getFormattedEmbargoDate(Dataset dataset) {
+        String embargoDateFormatted = StringUtils.EMPTY;
+        Date embargoDate = dataset.getEmbargoDate().getOrNull();
```

Comment:
It's easier to just utilize provided stream structure like `return dataset.getEmbargoDate().map(date -> dateFormatter.format(date)).getOrElse(StringUtils.EMPTY);`